examples/ipv6-eks-cluster/main.tf

```diff
@@ -111,76 +118,37 @@ module "aws-eks-accelerator-for-terraform" {
   vpc_id             = module.aws_vpc.vpc_id
   private_subnet_ids = module.aws_vpc.private_subnets
 
+  # IPV6
+  cluster_ip_family = "ipv6"
+
   # EKS CONTROL PLANE VARIABLES
-  kubernetes_version = local.kubernetes_version
+  cluster_version = local.cluster_version
 
   # EKS MANAGED NODE GROUPS
   managed_node_groups = {
     mg_4 = {
-      node_group_name = "managed-ondemand"
-      instance_types  = ["m4.large"]
+      node_group_name = "mng-ondemand"
+      instance_types  = ["m5.large"]
+      min_size        = "2"
+      desired_size    = "2"
+      max_size        = "10"
       subnet_ids      = module.aws_vpc.private_subnets
     }
   }
 }
 
 module "kubernetes-addons" {
-  source = "../../modules/kubernetes-addons"
-
+  source         = "../../modules/kubernetes-addons"
   eks_cluster_id = module.aws-eks-accelerator-for-terraform.eks_cluster_id
 
-  # EKS Addons
-  enable_amazon_eks_vpc_cni = true # default is false
-  #Optional
-  amazon_eks_vpc_cni_config = {
-    addon_name               = "vpc-cni"
-    addon_version            = "v1.10.1-eksbuild.1"
-    service_account          = "aws-node"
-    resolve_conflicts        = "OVERWRITE"
-    namespace                = "kube-system"
-    additional_iam_policies  = []
-    service_account_role_arn = ""
-    tags                     = {}
-  }
-
-  enable_amazon_eks_coredns = true # default is false
-  #Optional
-  amazon_eks_coredns_config = {
-    addon_name               = "coredns"
-    addon_version            = "v1.8.4-eksbuild.1"
-    service_account          = "coredns"
-    resolve_conflicts        = "OVERWRITE"
-    namespace                = "kube-system"
-    service_account_role_arn = ""
-    additional_iam_policies  = []
-    tags                     = {}
-  }
+  # EKS Managed Add-ons
+  enable_ipv6                  = true
```

Comment:
is this really an add-on? Or is it just part of cluster config? Like cluster version?

Reply:
No its not an add-on at all. I will update the example to make it more clear. 
This variable is used to setup additional policies for VPC CNI